rfcs/000-ml-feature-discovery.md

```diff
@@ -0,0 +1,168 @@
+- Feature Name: ml_feature_discovery
+- Start Date: 2021-05-05
+- RFC PR: [amundsen-io/rfcs#0000](https://github.com/amundsen-io/rfcs/pull/0000) (after opening the RFC PR, update this with a link to it and update the file name)
```

Comment:
```suggestion
- RFC PR: [amundsen-io/rfcs#36](https://github.com/amundsen-io/rfcs/pull/36)
```

Reply:
did this change?

Reply:
I clicked commit change, this is weird I'll try again

Reply:
done!

Reply:
okay now done for real hahaha